### PR TITLE
Fixes for #8 & #9

### LINKED ISSUE #8: Install issue: quickstart.sh has no preflight check for availability of wget
Just tried the quickstart on a new mac and it failed midway through because wget was missing.
Looks like the script could use an additional pre-flight check

### PR DESCRIPTION
**Pull request by wrigleyDan**
This PR is meant to fix https://github.com/querqy/chorus-elasticsearch-edition/issues/8 & https://github.com/querqy/chorus-elasticsearch-edition/issues/9

Issues both affected the `quickstart.sh` script:
- product data is now written to a file not dumped to the console.
- pre-flight check for wget is now included.

### CODE REVIEW

**Review by epugh — APPROVED** [review]
LGTM

**Comment by epugh** [comment]
@wrigleyDan do you have the access to redeploy the demo?   ANything else to get these PR's merged and tickets closed?

**Comment by wrigleyDan** [comment]
@epugh I can redeploy the demo but I have no write access to get these PRs merged and the tickets closed. So once the PRs are merged I can update our demo.

**Pull request merged**
